apps/ledger-live-mobile/src/screens/Account/index.tsx

```diff
@@ -50,7 +54,22 @@ const AnimatedFlatListWithRefreshControl = Animated.createAnimatedComponent(
 /** If deep linking params are present, this Account Screen is redirected to from Accounts Screen. */
 function AccountScreen({ route }: Props) {
   const { account, parentAccount } = useSelector(accountScreenSelector(route));
-
+  const navigation =
+    useNavigation<RootNavigationComposite<StackNavigatorNavigation<BaseNavigatorStackParamList>>>();
+
+  useEffect(() => {
+    if (!account) {
+      navigation.navigate(NavigatorName.Base, {
+        screen: NavigatorName.Main,
+        params: {
+          screen: NavigatorName.Portfolio,
+          params: {
+            screen: NavigatorName.WalletTab,
+          },
+        },
+      });
+    }
+  }, [account, navigation]);
   if (!account) return null;
```

Comment:
question: would you keep this ligne  while you already navigate to another screen (so this component will explicitly unmount) in the useEffect ?

Reply:
`Type 'AccountLike | undefined' is not assignable to type 'AccountLike'.
  Type 'undefined' is not assignable to type 'AccountLike'.`
  
  if I remove it